foundry.md

```diff
@@ -17,13 +19,57 @@ endmodule
 Foundry Success Predicate
 -------------------------
 
+```k
+module FOUNDRY-ACCOUNTS
+    imports SOLIDITY-FIELDS
+
+    syntax Contract        ::= FoundryContract
+    syntax Field           ::= FoundryField
+    syntax FoundryContract ::= "Foundry"      [klabel(contract_Foundry)]
+                             | "FoundryTest"  [klabel(contract_FoundryTest)]
+                             | "FoundryCheat" [klabel(contract_FoundryCheat)]
+    syntax FoundryField    ::= "Failed"       [klabel(slot_failed)]
+ // -------------------------------------------------------------------------
+    rule #address(Foundry)      => 137122462167341575662000267002353578582749290296  // 0x1804c8AB1F12E6bbf3894d4083f33e07309d1f38
+    rule #address(FoundryTest)  => 1032069922050249630382865877677304880282300743300 // 0xb4c79daB8f259C7Aee6E5b2Aa729821864227e84
+    rule #address(FoundryCheat) => 645326474426547203313410069153905908525362434349  // 0x7109709ECfa91a80626fF3989D68f67F5b1DD12D
+    rule #loc(FoundryCheat . Failed) => 46308022326495007027972728677917914892729792999299745830475596687180801507328
+    //0x6661696c65640000000000000000000000000000000000000000000000000000
+
+endmodule
+```
+The Foundry success predicate performs the same checks that the `is_success` function from Foundry in `evm/src/executor/mod.rs`.
```

Comment:
`````suggestion
```

The Foundry success predicate performs the same checks that the `is_success` function from Foundry in `evm/src/executor/mod.rs`.
`````